platform/authorized_api/apps/index.json

```diff
@@ -337,7 +345,7 @@
         },
         "privacyCookieDomains": {
           "type": "array",
-          "description": "List of privacy cookie domains for an app (works if enabled)",
+          "description": "List of privacy cookie domains for an app (works if enabled), you can also use wildcards like *.example.com",
```

Comment:
It's used when use consent (privacy settings) are shared by multiple domains (e.g. `blog.example.com` and `shop.example.com`)? It may be worth mentioning that.